legal-api/report-templates/template-parts/certificate-of-name-change/seal.html

```diff
@@ -0,0 +1,3 @@
+<div class="certificate-seal">
```

Comment:
Is this a duplicate of another seal? If so, can it be made common?

Reply:
Yes, you're right. I'm going to move to common. Thanks.

Reply:
And update any other reports that use the same one? :)

Reply:
yes: certificateOfIncorporation. I did already a change to certificateOfIncorporation to make the styleCertificate.css a common style among certificates.